pkg/notify/notify.go

```diff
@@ -135,11 +97,55 @@ func (n *Notifier) Close() {
 	n.mu.Lock()
 	defer n.mu.Unlock()
 	for _, receiver := range n.receivers {
-		if receiver.rec.ticker != nil {
-			receiver.rec.ticker.Stop()
-		}
-		close(receiver.rec.closeCh)
+		receiver.rec.close()
 	}
 	n.receivers = nil
 	n.closed = true
 }
+
+// Receiver is a receiver of notifier, including the receiver channel and stop receiver function.
+type Receiver struct {
+	C       <-chan struct{}
```

Comment:
It seems the `C` and `c` are identical, can we eliminate the redundancy one?
Ref: https://github.com/pingcap/tiflow/pull/7077/files#diff-d978136a23c5c118a3dc8719ba1abeeb49bb025f44d4d09555602128ab4f79a9R59-R67